docs/topics/cache.txt

```diff
@@ -891,6 +891,17 @@ from the cache, not just the keys set by your application. ::
 
     >>> cache.clear()
 
+.. versionchanged:: 1.11
+
+    The ``cache.touch()`` method was added.
+
+You can prolong/update the timeout value of an entry using ``cache.touch()``. ::
+
+    >>> cache.touch('a')
+
+Like other methods, ``touch()`` takes an optional ``timeout`` parameter. ``touch()``
```

Comment:
I think that this should explain how `timeout` is applied, i.e. it's re-applied to be from *now*, not when the key was initially put into the cache